src/app/ui/components/containers/footers/footer.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex } from 'leather-styles/jsx';
+
+import { HasChildren } from '@app/common/has-children';
+
+/** Footer is only used at smaller widths - in Dialog / Page card view / extension size */
```

Comment:
I looked into adding better documentation to the stories but couldn't configure it easily. 

I realised then that if you just add JSDOCs syntax like this it will show up in the story so you can easily add some notes there:
```
/** Some basic docs */
```
![Screenshot 2024-01-19 at 06 20 19](https://github.com/leather-wallet/extension/assets/2938440/2ef97a7b-6088-4042-81d4-e753724a19a8)